commands/bmc.vegman

```diff
@@ -304,6 +304,7 @@ function cmd_config_default {
     esac
     shift
   done
+  log_audit "Restore the factory default BMC config"
```

Comment:
Why don't you log the `datetime method`, `datetime set`, `datetime ntpconfig`, `ifconfig`, and `syslog` configuration commands?

For `ifconfig` and `datetime ntpconfig` I think the logging must be done inside `netconfig`.